tests/qa_automation/kernel_multipath_flakyserver.pm

```diff
@@ -0,0 +1,116 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: run multipath (I/O) tests on a multipathed iSCSI device while
+#          the iSCSI server keeps tampering with the LUNs (robustness test)
+# Maintainer: Klaus G. Wagner <kgw@suse.com>
+
+use base 'qa_run';
+use strict;
+use warnings;
+use lockapi;
+use testapi;
+use mmapi;
+use utils;
+use iscsi;
+
+sub start_testrun {
+    my $self = shift;
+
+    my $count;
+    # Turns out this provides the job ID of the one parallel job
+    # _NOTE_: mutex_wait(), mutex_lock(), mutex_unlock(), mutex_try_lock() **need** this as second arg!
+    my $jobid_server = get_children();
+    my $jobid_server = get_parents();
```

Comment:
Are you sure that you want to use `$jobid_server` for both, calls? (get_parents and get_children)

Reply:
Argh, copy-and-paste error. Thanks for spotting it! Of course, the get_children() line is unneeded and will be deleted.

Reply:
As of now I'm unfortunately unable to provide new verification runs for my amendment due to some generic openQA mess-up:
From http://polya.suse.de/tests/468/file/serial_terminal.txt
```
# zypper -n in qa_testset_automation qa_tools python-base python-xml 
# .....
...
Package 'python-xml' not found.

```
So things have to wait, sorry.

Reply:
@klwagner please get newer image, on 15sp1 it's using python2 module to solve issue with missing python-xml e.g. https://openqa.suse.de/tests/3404499

Reply:
I meanwhile found references (partly by you) regarding the analogous problem for the qa_kernel_multipath test:

https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6878/ (--> commit  b7a16a9b)
https://progress.opensuse.org/issues/54944
https://progress.opensuse.org/issues/48308

I'll do as you said and rerun my job from a more recent system disk image. HDD_1 from http://polya.suse.de/tests/468/#settings (dated 20190726) looks indeed too old in view of the above.

Reply:
A recent disk image for the client did indeed work.

Fixed PR in amended commit e74cff95 and provided new verification runs (see updated references in the top comment).

Reply:
That's great, just please fix the openQA tidy, with make test or manually.
```
-        foreach (1,2,3,4,5) {
+        foreach (1, 2, 3, 4, 5) {
```

Reply:
Done in the next amended commit. Anyway, this code is superseded by the latest amended commits.